### DISCUSSION TITLE
Why is it free?! and question about microsoft speech synthesizer

### DISCUSSION THREAD

**Discussion by nielka98** [SubtitleEdit/subtitleedit#8463] [General]
First of all I want to thank you for adding the possibility of generating a narrator  I love you for that because not only can I generate subtitles for free but also a narrator based on these subtitles There is nothing better Nicolas thank you thank you and you are crazy That you make it available for free
and now I would like to come to the issue that caused Nikola to write expensively at all, is it possible to add voices to the microsoft speech synthesizer engine manually and if not, are you planning to add more voices in the next updates?

**Comment by niksedk** [comment]
You can refresh voices by right-click in the voice combo box:
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/63631bbc-b96d-4b41-846d-75f8fb4d9420)


(note, you need an API key for online services)

**Comment by nielka98** [comment]
But I'm not talking about Microsoft Azure, just the normal one

**Comment by niksedk** [comment]
I do not know how to get more voices for the old microsoft speech synthesizer engine, but I recommend that you try Piper or "All Talk".

**Comment by nielka98** [comment]
How do I install because  All Talk I keep getting errors

**Comment by niksedk** [comment]
See installation instructions for stand alone here: https://github.com/erew123/alltalk_tts/#-quick-setup-text-generation-webui--standalone-installation

**Reply by smalhotr85** [reply]
thank nik, can i use indian languages(hindi, marathi, tamil etc) if use microsoft speech synthesizer(veryfast/robotic)? if yes, please share some link which can guide me.

**Comment by Nikhil-sharma-c** [comment]
phonemizer==3.3.0 or misaki or espeak can be used to generate indian languages .
best option will be espeak as it come ready made only you have to integrate with it .
